GUI/src/main/java/ui/UIController.java

```diff
@@ -594,6 +628,12 @@ public static void clearJobHistory() {
         Called from the Edit menu, this wipes out the job history that is stored in the users file system
          */
         INSTANCE.getHistory().clear();
+        try {
+            DbConnection dbConnection = DbConnection.getInstance();
+            dbConnection.deleteFilesHistory();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
```

Comment:
We can show a popup mesage using `ConfirmationDialog` class instead of throwing `RuntimeException`. What do you think?